masterでRuby3.1にしたところ、Turboのキャッシュによりテストが落ちたり落ちなかったりする

PR #443 から発生。

## CIが落ちた様子

酒のコピーだけテストが落ちる

```console
DEBUGGER: Attaching after process 3521 fork to child process 3699
........................................................................2022-03-19 13:31:50 WARN Selenium [DEPRECATION] [:browser_options] :options as a parameter for driver initialization is deprecated. Use :capabilities with an Array of value capabilities/options if necessary instead.
F..........................................................................................................................................................................

Failures:

  1) CopySakes copied values kura and todofuken has copied kura and todofuken
     Failure/Error: expect(find(:test_id, id).value).to have_text("#{sake.kura}（#{sake.todofuken}）")
       expected to find text "原田酒造合資会社（愛知県）" in ""

     [Screenshot Image]: /home/runner/work/sakazuki/sakazuki/tmp/capybara/failures_r_spec_example_groups_copy_sakes_copied_values_kura_and_todofuken_has_copied_kura_and_todofuken_718.png


     # ./spec/system/copy_sakes_spec.rb:90:in `block (4 levels) in <top (required)>'

Finished in 1 minute 10.44 seconds (files took 15.11 seconds to load)
243 examples, 1 failure

Failed examples:

rspec ./spec/system/copy_sakes_spec.rb:89 # CopySakes copied values kura and todofuken has copied kura and todofuken

Stopped processing SimpleCov as a previous error not related to SimpleCov has been detected
Coverage report generated for RSpec to /home/runner/work/sakazuki/sakazuki/coverage. 320 / 342 LOC (93.57%) covered.
Error: Process completed with exit code 1.
```

## 手元だと違うテストが落ちる

- 再現性なくあちこち落ちる

```console
$ rspec
..................FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFDEBUGGER: Attaching after process 21871 fork to child process 22726
2022-03-21 18:48:10 WARN Selenium [DEPRECATION] [:browser_options] :options as a parameter for driver initialization is deprecated. Use :capabilities with an Array of value capabilities/options if necessary instead.
.........................................................................................

Failures:

  1) Photo chackbox_name
     Failure/Error: let(:photo) { create(:photo) }

     Faraday::ConnectionFailed:
       Connection reset by peer
     # <internal:kernel>:90:in `tap'
     # <internal:kernel>:90:in `tap'
     # ./spec/models/photo_spec.rb:17:in `block (3 levels) in <top (required)>'
     # ./spec/models/photo_spec.rb:15:in `block (3 levels) in <top (required)>'
     # ./spec/models/photo_spec.rb:19:in `block (3 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # Errno::ECONNRESET:
     #   Connection reset by peer
     #   <internal:kernel>:90:in `tap'

  2) Sake validates if name is missing
     Failure/Error: let!(:sealed_sake) { create(:sake, bottle_level: "sealed", size: 720) }

     Faraday::ConnectionFailed:
       end of file reached
     # <internal:kernel>:90:in `tap'
     # ./spec/models/sake_spec.rb:46:in `block (2 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # EOFError:
     #   end of file reached
     #   <internal:kernel>:90:in `tap'
以下省略
```

## 他にも、flashメッセージが表示されないなど不具合がある

ログイン後のログインメッセージがでない

![image](https://user-images.githubusercontent.com/849256/159237589-4a82f8de-1d9f-4cfa-a673-ae9348bddb62.png)

--- a/Gemfile
+++ b/Gemfile
@@ -19,7 +19,7 @@ gem "puma"
 gem "jsbundling-rails"
 
 # Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
-gem "turbo-rails"
+# gem "turbo-rails"
 
 # Hotwire's modest JavaScript framework [https://stimulus.hotwired.dev]
 # gem "stimulus-rails"
@@ -31,7 +31,7 @@ gem "cssbundling-rails"
 gem "jbuilder"
 
 # Use Redis adapter to run Action Cable in production
-gem "redis"
+# gem "redis", "~> 4.0"
 
 # Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
 # gem "kredis"

--- a/app/helpers/application_helper.rb
+++ b/app/helpers/application_helper.rb
@@ -48,7 +48,7 @@ module ApplicationHelper
   # @param name [String] 読み込む.jsファイル名、複数記述できる
   def select_js(*name)
     content_for(:js) do
-      javascript_include_tag(*name, "data-turbo-track": "reload", defer: true)
+      javascript_include_tag(*name, defer: true)
     end
   end
 

--- a/app/helpers/application_helper.rb
+++ b/app/helpers/application_helper.rb
@@ -57,7 +57,7 @@ module ApplicationHelper
   # @param name [String] 読み込む.scssファイル名、複数記述できる
   def select_css(*name)
     content_for(:css) do
-      stylesheet_link_tag(*name, "data-turbo-track": "reload")
+      stylesheet_link_tag(*name)
     end
   end
 

--- a/app/javascript/application.js
+++ b/app/javascript/application.js
@@ -1,5 +1,4 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
 import Rails from "@rails/ujs"
 Rails.start()
 

--- a/app/views/layouts/application.html.erb
+++ b/app/views/layouts/application.html.erb
@@ -6,7 +6,7 @@
     <%= csp_meta_tag %>
     <%= display_meta_tags(default_meta_tags) %>
     <%= adsense_tags %>
-    <%= stylesheet_link_tag("application", media: "all", "data-turbo-track": "reload") %>
+    <%= stylesheet_link_tag("application", media: "all") %>
     <%= yield(:css) if content_for?(:css) %>
     <%= javascript_include_tag("application", "data-turbo-track": "reload", defer: true) %>
     <%= yield(:js) if content_for?(:js) %>

--- a/app/views/layouts/application.html.erb
+++ b/app/views/layouts/application.html.erb
@@ -8,7 +8,7 @@
     <%= adsense_tags %>
     <%= stylesheet_link_tag("application", media: "all") %>
     <%= yield(:css) if content_for?(:css) %>
-    <%= javascript_include_tag("application", "data-turbo-track": "reload", defer: true) %>
+    <%= javascript_include_tag("application", defer: true) %>
     <%= yield(:js) if content_for?(:js) %>
   </head>
 

--- a/config/cable.yml
+++ b/config/cable.yml
@@ -1,6 +1,5 @@
 development:
-  adapter: redis
-  url: redis://localhost:6379/1
+  adapter: async
 
 test:
   adapter: test

--- a/package.json
+++ b/package.json
@@ -2,7 +2,6 @@
   "name": "sakazuki",
   "private": true,
   "dependencies": {
-    "@hotwired/turbo-rails": "^7.1.1",
     "@rails/ujs": "7.0.2-2",
     "bootstrap": ">=5.1.3 <6",
     "bootstrap-icons": ">=1.8.1 <2",

